content/zh/docs/contribution_guidelines/conduct.md

```diff
@@ -0,0 +1,166 @@
++++ 
+title = "行为规范"
+weight = 480 
++++
+
+<h2 id="about">关于本规范</h2>
+
+<p>
+线上社区会有很多不同背景的人，CUE 贡献者致力于为所有人提供一个友好、安全以及热情的环境，
+不论性别、性取向、是否健全、智力、外貌、身高、种族、国籍、人种、年龄、信仰或其他个人特征。
+</p>
+
+<p>
+行为规范的第一个目的是制定行为的基本标准，这样不同价值观以及不同沟通风格的人们可以有效、高效以及互相尊重地讨论 CUE。
+</p>
+
+<p>
+第二个目的是当社区中发生冲突时提供解决冲突的机制。
+</p>
+
+<p>
+第三个目的是让我们的社区对来自不同背景的人都保持欢迎态度。
+多向性对这个项目至关重要；为了成功，CUE 需要来自不同背景的贡献者和用户。（启发于 <a href="https://blog.golang.org/open-source">Go, Open Source, Community</a>）
+</p>
+
+<p>
+我们相信正向的争论或分歧对于项目以及社区至关重要。但是，不尊重人是绝对不行的。我们重视不同的观点，但是我们重视有礼貌的行为。
+</p>
+
+<h2 id="values">CUE 的价值观</h2>
+
+<p>
+这些是 CUE 社区中应该尊重的价值观。
+</p>
+
+<ul>
+<li>保持友好以及热情
+<li>保持耐心
+    <ul>
+        <li>请谨记人们有不同的交流方式而且也不是所有人都用同一种语言。（话语本身的意思或语气可能会在翻译中丢失）
+    </ul>
+<li>考虑周全
+    <ul>
+    <li>高效的沟通需要多用心，多考虑你的话怎么才能比较好理解。
+    <li>记住有时候不评论是最好的选择。
+    </ul>
+<li>保持尊重
+    <ul>
+    <li>特别是要尊重不同的意见
+    </ul>
+<li>乐于助人
+    <ul>
+    <li>真诚地解释其他人的观点，不要吹毛求疵
+    <li>当确实有不同观点时，要尽量解释原因
+    </ul>
+<li>避免以下行为：
+    <ul>
+    <li>跑题：专注于主题；如果你想要讨论其他事情，请重开新的对话
+    <li>非建设性批评：不要只是谴责当前事态；提供或至少征求改进的建议
+    <li>吐槽(简短、低效、带有恶意的评论)
+    <li>讨论可能冒犯或敏感的问题；这些通常会导致不必要的争执
+    <li>微歧视：口头、行为以及环境的侮辱，传达敌意、贬低或否定的信息对个人或团体进行轻视或侮辱。
+    </ul>
+</ul>
+
+<p>
+人是非常复杂的，肯定会被误解或误解别人。当这些不可避免的冲突发生时，请克制出于自我保护的冲动或推卸责任。
+也请不要挑起无端的冲突，要先假定对方是无意冒犯。即使对方故意挑衅，也不要将争端升级。
+当冲突发生时，<i>所有人</i>都有责任将冲突降级。
+</p>
+
+<h2 id="code">行为规范</h2>
+
+<h3 id="our-pledge">我们的承诺</h3>
+
+<p>
+为了维护一个开放和有好的环境，我们作为贡献者和维护者承诺，项目和社区中的所有人都可以随时提问，
```

Comment:
Again with reference to DeepL (apologies again for that being my only reference here!) I'm not sure that the start of this very long sentence conveys what we need it to. The original reads:

> In the interest of fostering an open and welcoming environment, we as contributors and maintainers pledge to making participation in our project and our community a harassment-free experience for everyone, regardless of ...

The [DeepL translation](https://www.deepl.com/en/translator#zh/en/%E4%B8%BA%E4%BA%86%E7%BB%B4%E6%8A%A4%E4%B8%80%E4%B8%AA%E5%BC%80%E6%94%BE%E5%92%8C%E6%9C%89%E5%A5%BD%E7%9A%84%E7%8E%AF%E5%A2%83%EF%BC%8C%E6%88%91%E4%BB%AC%E4%BD%9C%E4%B8%BA%E8%B4%A1%E7%8C%AE%E8%80%85%E5%92%8C%E7%BB%B4%E6%8A%A4%E8%80%85%E6%89%BF%E8%AF%BA%EF%BC%8C%E9%A1%B9%E7%9B%AE%E5%92%8C%E7%A4%BE%E5%8C%BA%E4%B8%AD%E7%9A%84%E6%89%80%E6%9C%89%E4%BA%BA%E9%83%BD%E5%8F%AF%E4%BB%A5%E9%9A%8F%E6%97%B6%E6%8F%90%E9%97%AE%EF%BC%8C%0A%E6%97%A0%E8%AE%BA%E5%B9%B4%E9%BE%84%E3%80%81%E4%BD%93%E5%9E%8B%E3%80%81%E6%98%AF%E5%90%A6%E5%81%A5%E5%85%A8%E3%80%81%E7%A7%8D%E6%97%8F%E3%80%81%E6%80%A7%E5%88%AB%E3%80%81%E7%BB%8F%E9%AA%8C%E3%80%81%E5%8F%97%E6%95%99%E8%82%B2%E7%A8%8B%E5%BA%A6%E3%80%81%E7%A4%BE%E4%BC%9A%E7%BB%8F%E6%B5%8E%E5%9C%B0%E4%BD%8D%E3%80%81%E5%9B%BD%E7%B1%8D%E3%80%81%E5%A4%96%E8%B2%8C%E3%80%81%E7%A7%8D%E6%97%8F%E3%80%81%E4%BF%A1%E4%BB%B0%E4%BB%A5%E5%8F%8A%E6%80%A7%E5%8F%96%E5%90%91%E3%80%82) appears to show:

> In order to maintain an open and supportive environment, we as contributors and maintainers are committed to the project and to the community where all people can ask questions at any time.

The original is quite a bit stronger and broader in an important sense:

> ... making participation in our project and our community a harassment-free experience for everyone

Is there a tweak possible here?

Reply:
I misunderstand the word `harassment-free`. 
I know it, it means do not harass others rather than feel free to harass others.